src/peerstate.rs

```diff
@@ -362,6 +362,13 @@ impl Peerstate {
                 self.public_key = Some(header.public_key.clone());
                 self.recalc_fingerprint();
             }
+        } else {
+            warn!(
```

Comment:
Btw, we can print some debug logs only in the `test` configuration, if we don't want to clutter release configuration logs. Not sure about this one though